backend/tests/v5/unit/models/revisions.test.js

```diff
@@ -53,7 +53,75 @@ const testGetLatestRevision = () => {
 	});
 };
 
+const testGetRevisions = () => {
+	describe('GetRevisions', () => {
+		const expectedData = [
+			{ _id: 1, author: 'someUser', timestamp: new Date() },
+			{ _id: 2, author: 'someUser', timestamp: new Date() },
+			{ _id: 3, author: 'someUser', timestamp: new Date(), void: true },
+		];
+
+		test('Should return non void container revisions', async () => {
+			const nonVoidData = expectedData.filter((d) => !d.void);
+			jest.spyOn(db, 'find').mockResolvedValue(nonVoidData);
+			const res = await Revisions.getRevisions('someTS', 'someModel', false);
+			expect(res).toEqual(nonVoidData);
+		});
+
+		test('Should return all container revisions', async () => {
+			jest.spyOn(db, 'find').mockResolvedValue(expectedData);
+			const res = await Revisions.getRevisions('someTS', 'someModel', true);
+			expect(res).toEqual(expectedData);
+		});
+
+		test('Should throw CONTAINER_NOT_FOUND if there is no revisions table', async () => {
+			jest.spyOn(db, 'find').mockResolvedValue({ length: 0 });
+			await expect(Revisions.getRevisions('someTS', 'someModel')).rejects.toEqual(templates.containerNotFound);
+		});
+	});
+};
+
+const testUpdateRevisionStatus = () => {
+	describe('UpdateRevisionStatus', () => {
+		const expectedData = [
+			{ _id: 1, author: 'someUser', timestamp: new Date() },
+			{ _id: 2, author: 'someUser', timestamp: new Date() },
+			{ _id: 3, author: 'someUser', timestamp: new Date(), void: true },
+		];
+
+		test('Should update the void status of a revision', async () => {
+			jest.spyOn(db, 'find').mockResolvedValue(expectedData);
+			await Revisions.updateRevisionStatus('someTS', 'someModel', 3, false);
+			expect(expectedData.find((r) => r._id === 3).void).toEqual(false);
+		});
+
+		test('Should not update the void status of a revision if void is not set and new status is false', async () => {
```

Comment:
this test is redundant i think?

After the change were we stop probing what was the last state and just set/unset the void status regardless, it's not really testing anything new to the previous test.